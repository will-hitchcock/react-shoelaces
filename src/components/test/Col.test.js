import Col from '../Col'

describe('container', () => {
  it('should render a <Col />', () => {
    expect(shallow(<Col />)).toMatchSnapshot()
  })
})
