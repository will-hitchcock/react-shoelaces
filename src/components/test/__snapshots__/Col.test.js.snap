// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`container should render a <Col /> 1`] = `
<div
  className="sc-bdVaJa eOlpUT"
/>
`;
